Gruntfile.js

```diff
@@ -219,7 +219,6 @@ module.exports = function (grunt) {
         src: 'softwerkskammer/testApp',
         options: {
           coverageFolder: 'softwerkskammer/coverageApp',
-          excludes: ['**/activitystore.js'],
```

Comment:
Da _muss_ man es excluden

> Am 06.08.2016 um 23:18 schrieb Nicole Rauch <notifications@github.com>:
> 
> In Gruntfile.js <https://github.com/softwerkskammer/Agora/pull/1099#discussion_r73792137>:
> 
> > @@ -219,7 +219,6 @@ module.exports = function (grunt) {
> >          src: 'softwerkskammer/testApp',
> >          options: {
> >            coverageFolder: 'softwerkskammer/coverageApp',
> > -          excludes: ['**/activitystore.js'],
> Hängt es vielleicht damit zusammen, dass der code in den anderen Tests (mit DB) schon erfasst wird?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/softwerkskammer/Agora/pull/1099/files/53cf2a557804c94cffd2634ad90942854bcfa82b#r73792137>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAbglAAQEPZ_ODPyLP1i4Nt0P9EycK6Mks5qdPoLgaJpZM4JeVWa>.
>